extension/public/manifest.json

```diff
@@ -44,5 +45,11 @@
       ],
       "matches": ["*://*.x.com/*", "*://x.com/*"]
     }
-  ]
+  ],
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "unbaited@example.com",
```

Comment:
Should probably set this to something reasonable. Setting it is necessary for the storage API to work in Firefox (at least in development mode).